torch/fx/experimental/symbolic_shapes.py

```diff
@@ -538,16 +538,17 @@ def wrapper(self, *args, **kwargs):
     # NB: This inherits from Dummy, not Symbol, because Symbols with the same
     # name get interned.  This is bad for us as we want the metadata
     # to vary across different invocations and not leak.
-    class Symbol(sympy.Dummy):
```

Comment:
why?

Reply:
The Symbol subclass is very useful and I'd like to keep it haha

Reply:
Yeah I think we should be able to. I was just tracking down an issue where sympy substitution wasn't working, and it was because one of the symbols was this one and the other one was a normal `sympy.Symbol` >:(